.changeset/mean-peas-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/react': minor
+---
+
+Box: Added breakWords prop to allow words to break.
```

Comment:
```suggestion
Box: Added breakWords prop to allow words to break across two lines.
```

Reply:
changed it to say "across multiple lines" instead of 2 lines since it is possible for a word to break across 3 lines.